src/map.cpp

```diff
@@ -5086,43 +5074,44 @@ std::list<std::pair<tripoint, item *> > map::get_rc_items( int x, int y, int z )
     return rc_pairs;
 }
 
-static bool trigger_radio_item( item_stack &items, item_stack::iterator &n,
+static bool trigger_radio_item( item_stack &items, safe_reference<item> &safe_ref,
```

Comment:
It seems unlikely this needs a `safe_reference`.  Can it just take an `item*` or `item&`?

Reply:
It needs to be a
```cpp
        using map_process_func = bool ( * )( item_stack &, safe_reference<item> &, const tripoint &,
                                             const std::string &, float, temperature_flag );
```
so unfortunately not

Reply:
Understood.